### PR TITLE
Revert "Revert "Add a section for building docs""

### PR DESCRIPTION
**Pull request by lubosmj**
This reverts commit a30c5aaf3f3479983ec0bd663cdbb9e15b5dacae and fixes the docs port exposed to the world.

### CODE REVIEW

File: client/oci_env/utils.py [hunk 1]
```diff
@@ -71,6 +71,10 @@ def get_config(env_file):
         "API_PROTOCOL": "http",
         "API_HOST": "localhost",
         "API_PORT": "5001",
+
+        # A port dedicated for exposing generated docs
+        "DOCS_PORT": "12345",
+        "NGINX_DOCS_PORT": user_preferences.get("DOCS_PORT", "12345"),
```
Comment (lubosmj) [inline client/oci_env/utils.py:77]: This is trying to address the comment from https://github.com/pulp/oci_env/pull/102#discussion_r1151032579.

Comment (decko) [reply client/oci_env/utils.py:77]: With this the ports will not collide when running two oci-envs?

**Review by newswangerd — APPROVED** [review]

**Comment by newswangerd** [comment]
@lubosmj I'm gonna create a PR to fix the galaxy_ng ci for this change, then I'll merge this.

**Comment by lubosmj** [comment]
Is it still failing? Weird...

**Comment by newswangerd** [comment]
@lubosmj I think it will when this pr is merged. I have to update the sync test on our ci to use different ports for the docs service.

**Comment by lubosmj** [comment]
Okay, please use the `NGINX_DOCS_PORT` variable that is supposed to behave like `NGINX_PORT`.

**Comment by lubosmj** [comment]
@newswangerd, any update on your effort? Can we merge this PR?

**Pull request merged**